index.html

```diff
@@ -1027,6 +1027,11 @@ <h4>
             subsequent field value to the combined field value in order,
             separated by a comma.
           </p>
+          <p>
+            The user agent MAY still enforce cross-origin restrictions and set
```

Comment:
This goes a bit beyond what we previously discussed in the WG (I think we focused on the sizes attributes and unrestricted content-type). Are you OK with language that focuses on these attributes?

Reply:
Sure